monitoring/monitoring-overview.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+[id="about-openshift-container-platform-monitoring"]
```

Comment:
You could update this ID to be distribution agnostic, such as "about-openshift-monitoring". There could be use cases where this is reused in the future for OpenShift distributions other than OCP.